packages/select/custom-resources/AvOrganizationSelect.js

```diff
@@ -8,11 +8,16 @@ const OrganizationSelect = ResourceSelect.create({
   labelKey: 'name',
 });
 
-const AvOrganizationSelect = ({ name, resourceIds, ...props }) => {
+const AvOrganizationSelect = ({
+  name,
+  resourceIds,
+  permissionIds,
+  ...props
+}) => {
   return (
     <OrganizationSelect
       name={name}
-      additionalPostGetArgs={resourceIds ? { resourceIds } : undefined}
+      additionalPostGetArgs={{ resourceIds, permissionIds }}
```

Comment:
Need to add a check for `undefined` here otherwise `additionalPostGetArgs` will always get created and users won't be able to use the normal `postGet`
`additionalPostGetArgs={resourceIds || permissionIds ? { resourceIds, permissionIds } : undefined}`

Reply:
Added  `if (additionalPostGetArgs && additionalPostGetArgs.resourceIds)` to the `sdk-js` code, but could add this too

Reply:
I think it'd be helpful here too so that [cacheUniq](https://github.com/Availity/availity-react/blob/master/packages/select/src/ResourceSelect.js#L43) doesn't end up with an object containing undefined values, but I haven't tested to see if that's functionally different